charm/src/charm.ts

```diff
@@ -372,7 +376,7 @@ export class CharmManager {
   }
 
   async syncRecipeCells(recipeId: string) {
-    // NOTE(ja): I don't think this actually syncs the recipe
+    // NOTE(ja): this doesn't sync recipe to storage
     if (recipeId) await storage.syncCellById(this.space, { "/": recipeId });
```

Comment:
What _does_ it do...?

Reply:
it just creates a cell ... with nothing in it